### DISCUSSION TITLE
Our project only has 1/10 of 80*40 dot matrix screen. Do you have any plan to adapt this display recently

### DISCUSSION THREAD

**Discussion by ShiPC123** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#590] [General]
Our project only has 1/10 of 80*40 dot matrix screen. Do you have any plan to adapt this display recently

**Comment by mrcodetastic** [comment]
This library does not support these types of panels.

**Reply by ShiPC123** [reply]
Thank you very much. Will it be suitable later?

**Reply by board707** [reply]
You can try to use your panel with another library - [DMD_STM32](https://github.com/board707/DMD_STM32), but not with ESP32 boards.
The architecture of the DMD_STM32 library is written in such a way that adding support for hub75 panels of any resolution and scan factor requires editing just a couple of lines in the code.

**Reply by ShiPC123** [reply]
Thank you very much I will try it
